config/jobs/kubernetes/sig-network/sig-network-misc.yaml

```diff
@@ -265,6 +265,261 @@ presubmits:
+        resources:
+          limits:
+            cpu: 2
+            memory: "6Gi"
+          requests:
+            cpu: 2
+            memory: "6Gi"
+    annotations:
+      testgrid-dashboards: sig-network-gce
+      testgrid-tab-name: gce-kubedns-performance
+
+  - name: pull-kubernetes-e2e-gce-kubedns-performance-nodecache
+    cluster: k8s-infra-prow-build
+    branches:
+    # TODO(releng): Remove once repo default branch has been renamed
+    - master
+    - main
+    always_run: false
+    run_if_changed: '^(test/e2e/network/|cluster/addons/dns/kube-dns/)'
+    optional: true
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+    decorate: true
+    decoration_config:
+      timeout: 80m
+    spec:
+      containers:
+      - command:
+        - runner.sh
+        - /workspace/scenarios/kubernetes_e2e.py
+        args:
+        - --cluster=gce-kubedns-perf-cache
+        - --env=CLUSTER_DNS_CORE_DNS=false
+        - --env=KUBE_ENABLE_NODELOCAL_DNS=true
+        - --env=KUBE_ENABLE_NODE_PROBLEM_DETECTOR=none
+        - --extract=ci/latest
+        - --gcp-master-image=gci
+        - --gcp-node-image=gci
+        - --gcp-nodes=3
+        - --gcp-zone=us-east1-b
+        - --provider=gce
+        - --test_args=--ginkgo.focus=\[Feature:PerformanceDNS\]
+        - --timeout=60m
+        - --use-logexporter
+        image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20240515-17c6d50e24-master
+        resources:
+          limits:
+            cpu: 2
+            memory: "6Gi"
+          requests:
+            cpu: 2
+            memory: "6Gi"
+    annotations:
+      testgrid-dashboards: sig-network-gce
+      testgrid-tab-name: gce-kubedns-performance-nodecache
+
+  - name: pull-kubernetes-e2e-gci-gce-kube-dns
+    cluster: k8s-infra-prow-build
+    branches:
+    # TODO(releng): Remove once repo default branch has been renamed
+    - master
+    - main
+    always_run: false
+    run_if_changed: '^(test/e2e/network/|cluster/addons/dns/kube-dns/)'
+    optional: true
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+    decorate: true
+    decoration_config:
+      timeout: 170m
+    spec:
+      containers:
+      - command:
+        - runner.sh
+        - /workspace/scenarios/kubernetes_e2e.py
+        args:
+        - --check-leaked-resources
+        - --cluster=
+        - --env=CLUSTER_DNS_CORE_DNS=false
+        - --extract=ci/latest
+        - --gcp-zone=us-west1-b
+        - --ginkgo-parallel=30
+        - --provider=gce
+        - --test_args=--ginkgo.skip=\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|\[sig-storage\] --minStartupPods=8
+        - --timeout=150m
+        image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20240515-17c6d50e24-master
+        resources:
+          limits:
+            cpu: 2
+            memory: "6Gi"
+          requests:
+            cpu: 2
+            memory: "6Gi"
+
+  - name: pull-kubernetes-e2e-gci-gce-kube-dns-nodecache
```

Comment:
let's keep only this, and remove the `run_if_changed`

Reply:
Yeah, I was about to pick this as as well as it should cover both kubedns and nodecache.

Reply:
Done.